pkg/proxy/util/network.go

```diff
@@ -25,7 +25,7 @@ import (
 // for testing purposes.
 type NetworkInterfacer interface {
 	Addrs(intf *net.Interface) ([]net.Addr, error)
```

Comment:
@chenyw1990 as Tim says below, I think `Addrs` is no longer used, right?

Reply:
yes，I will delete it.